tests/modules/entity_embed_twig_test/entity_embed_twig_test.info.yml

```diff
@@ -0,0 +1,6 @@
+name: 'Entity Embed twig test'
```

Comment:
A little curious why we don't just merge this into entity_embed_test module itself?